### PR TITLE
Added Bootstrap, favicon, header image (with copyright notice in foot…

### PR DESCRIPTION
**Pull request by woodyaz**
I did some stuff.

* Added bootstrap
*  Favicon!
*  Stats is now an inline un-ordered list.
*  Header image?

Let me know what y'all think!

### CODE REVIEW

**Comment by EddyLuten** [comment]
Yes, I like. :feelsgood: But if you don't mind I'm popping in a few comments here and there.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    font-size: 1.2em;
+    font-family: "Roboto";
+}
+
+h1, h2, h3, h4, h5, h6 {
+    color: #EE7552;
+}
+
+.jumbotron {
+    background-image: url("../images/party.jpg");
```
Comment (EddyLuten) [inline css/style.css]: I love this image (and favicon), it really shows the kind of party I had in mind when making this. However, the image is licensed as CC BY-SA 4.0 and I'm not 100% sure if that's compatible with this site's MIT licensing.

Also, I can't comment on binary files, but did you make the favicon yourself or did you get it somewhere? If the former, also need to check if the licenses are compatible just to cover ass.

As far as style, do you think the image could be faded a bit or maybe some kind of gradient overlay? It's a bit hard to read the header text on top of it.

Lastly, and sorry for the waterfall of comments on this one thing, do you think it's possible to reduce the size of the image? It's 3.2MB and a bit on the sluggish side to load, even running locally.

network-load

File: index.html [hunk 2]
```diff
@@ -160,36 +132,46 @@
...
+        <h2>Town Name Randomizer</h2>
+        <p>
+          <button id="generate_town">Generate</button>
+        </p>
+        <p id="random_town">
+          <span class="prefix"></span><span class="suffix"></span>
+        </p>
+        -->
+
+    <footer class="text-center">
```
Comment (EddyLuten) [inline index.html]: Do you think the font size for the footer could be a bit smaller to differentiate it from the main content?

File: index.html [hunk 3]
```diff
@@ -1,122 +1,94 @@
...
-    div.stat strong {
-      cursor: help;
-      border-bottom: 1px dashed #333;
-    }
-    div.clear {
-      clear: both;
-    }
-    .error {
-      color: red;
-    }
```
Comment (EddyLuten) [inline index.html:40]: Consider adding the .error class back in since it provides feedback when a field is left empty or invalid.

**Review by EddyLuten — CHANGES_REQUESTED** [review]
Right direction, but a few things that need to be addressed before merging.

File: css/style.css (hunk 1 above)
Comment (woodyaz) [reply css/style.css]: Regarding the header image, that's a good question...  Let me do some digging and/or find something else.

The middle finger source is public domain.  https://commons.wikimedia.org/wiki/File:The_middle_finger.svg; I just made an icon from it.

I'll make the header more readable,, and I'll resize the header imagine, assuming there are no legal issues.

File: index.html (hunk 3 above)
Comment (woodyaz) [reply index.html:40]: The latest commit adds the 'has-error' class added from Bootstrap, which "reds" it all out.

**Comment by woodyaz** [comment]
I forgot to add, I changed the stats randomizer to make ti play better with bootstrap, but if there are any issues, I can change it back.

**Review by EddyLuten — APPROVED** [review]
👍 merging!

**Pull request merged**